histomicstk/segmentation/label/trace_boundary_cpp.cpp

```diff
@@ -1,18 +1,18 @@
 /*
 C++ version of ISBF for TraceBounds
 */
-#include "isbfcpp.h"
+#include "trace_boundary_cpp.h"
 
 #include <iostream>
 #include <list>
 #include <cmath>
 #include <vector>
 
-isbfcpp::isbfcpp()
+trace_boundary_cpp::trace_boundary_cpp()
 {
 }
 
-void isbfcpp::rot90(int nrows, int ncols, std::vector <std::vector<int> > matrix,
+void trace_boundary_cpp::rot90(int nrows, int ncols, std::vector <std::vector<int> > matrix,
                     std::vector <std::vector<int> > &matrix270,
                     std::vector <std::vector<int> > &matrix180,
                     std::vector <std::vector<int> > &matrix90)
```

Comment:
It is more modular to have rot90 take a matrix and output only one matrix that is rotated by 90 degrees.